lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -42,6 +45,8 @@ def initialize(options, multiple: false)
           @answer = nil
           @state = :root
           @multiple = multiple
+          @metadata_overdraw = false
```

Comment:
Can you add a small comment for what `metadata_overdraw` represents? I read the code and it isn’t clear to me.

Reply:
It probably needs a better variable name.  I had a hard time trying to figure out what to call it.

Basically, it's used as a flag to indicate that an extra line (in this case, the "metadata" line displayed by selecting or filtering) is present and the terminal output should be cleared between renders.  In other words, when `@metadata_overdraw` is true, add 1 to `num_lines` and call `clear_output` in the render loop.